### PR TITLE
Package gccjit.0.3.2

### PR DESCRIPTION
**Pull request by lukstafi**
### `gccjit.0.3.2`
Bindings to libgccjit, the GCC JIT compiler
`libgccjit` is an embeddable shared library being included in GCC for adding compilation to existing programs using GCC as the backend. This package organizes the API of `gccjit` into a few types and their corresponding modules.

---
* Homepage: https://github.com/lukstafi/ocaml-gccjit
* Source repo: git+https://github.com/lukstafi/ocaml-gccjit.git
* Bug tracker: https://github.com/lukstafi/ocaml-gccjit/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by lukstafi** [comment]
No luck, the FreeBSD CI targets fail. But `dune test` passes on my fresh installation of FreeBSD 14.

Let me know if I should disable the unsupported architectures.

**Comment by mseri** [comment]
We are trying to get back to cleanup the backlog but at the moment we are too stranded at our workplaces. I hope to get back to this soon. Not supporting bytecode is not a problem

File: packages/gccjit/gccjit.0.3.2/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "3.11"}
+  "base-unix"
+  "ctypes" {>= "0.14.0"}
+  "ctypes-foreign"
+  "conf-libgccjit"
+  "ppx_expect" {with-test & >= "v0.9.0"}
+  "odoc" {with-doc}
+]
+build: [
```
Comment (mseri) [inline packages/gccjit/gccjit.0.3.2/opam:23]:
```suggestion
conflicts: ["ocaml-option-bytecode-only"]
build: [
```

Comment (mseri) [reply packages/gccjit/gccjit.0.3.2/opam:24]: This solves the issues where only the bytecode compiler is present

Comment (mseri) [reply packages/gccjit/gccjit.0.3.2/opam:24]: For reference 
```
#=== ERROR while compiling gccjit.0.3.2 =======================================#
# context              2.2.0~alpha4~dev | linux/ppc64 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/lukstafi/ocaml-gccjit/archive/refs/tags/0.3.2.1.tar.gz)
# path                 ~/.opam/5.1/.opam-switch/build/gccjit.0.3.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p gccjit -j 175 @install
# exit-code            1
# env-file             ~/.opam/log/gccjit-6-e20b5f.env
# output-file          ~/.opam/log/gccjit-6-e20b5f.out
### output ###
# File "test/dune", line 8, characters 8-14:
# 8 |  (modes native))
#             ^^^^^^
# Error: this mode isn't available
```

**Comment by mseri** [comment]
Thanks and sorry for the long wait

**Pull request merged**